internal/highlight/highlight.go

```diff
@@ -450,9 +456,27 @@ func unhighlightLongLines(h string, n int) (string, error) {
 	return buf.String(), nil
 }
 
-// ParseLinesFromHighlight takes the highlighted html table and returns a slice of highlighted strings, where each string corresponds a single line in the original, highlighed file.
-func ParseLinesFromHighlight(input string) ([]string, error) {
-	doc, err := html.Parse(strings.NewReader(input))
+// Lines highlights the file and returns a list of highlighted lines.
+// The returned boolean represents whether or not highlighting was aborted due
+// to timeout.
+func Lines(ctx context.Context, p Params) ([]string, bool, error) {
```

Comment:
```suggestion
func CodeAsLines(ctx context.Context, p Params) ([]string, bool, error) {
```

I can imagine someone being confused that this requires the while file and "not just a few lines", so I suggest renaming to `CodeAsLines` instead.